apps/etterlatte-hendelser-pdl/src/main/kotlin/hendelserpdl/config/KafkaEnvironment.kt

```diff
@@ -0,0 +1,22 @@
+package no.nav.etterlatte.hendelserpdl.config
+
+import io.confluent.kafka.serializers.KafkaAvroDeserializer
+import io.confluent.kafka.serializers.KafkaAvroDeserializerConfig
+import no.nav.etterlatte.kafka.Kafkakonfigurasjon
+import org.apache.kafka.clients.consumer.ConsumerConfig
+import org.apache.kafka.common.IsolationLevel
+import java.util.*
+
+class KafkaEnvironment : Kafkakonfigurasjon<KafkaAvroDeserializer>(
+    groupId = "LEESAH_KAFKA_GROUP_ID",
+    deserializerClass = KafkaAvroDeserializer::class.java,
+    extra = {
```

Comment:
Er en del props her så hvorfor ikke en for hver av disse som er optional kontra generisk extra